.github/workflows/build-test.yml

```diff
@@ -92,6 +92,11 @@ jobs:
       with:
         command: license-check
 
+    - name: Build the bincode
```

Comment:
I put this in the `lint` section mainly because it already has the `cargo make` command. Should I break it out into its own section and make it dependent on the build? I couldn't decide. The `Install cargo-make` command takes a little while if I recall correctly from my memory work.

Reply:
I think I would make it its own job. #315 will allow us to share artifacts between jobs so that GitHub doesn't need to reinstall cargo-make several times.